### ISSUE TITLE
Additional MATE packages needed on ARM64/AARCH64 install

### ISSUE THREAD

**Issue by MahdiHedhli** [ParrotSec/alternate-install#6]
Deployed on both Debian Stretch and Ubuntu Bionic on Rock64 and RockPro64 SBCs.

Upon logging in to MATE the menu configuration was incorrect and I was prompted with the following errors:

The panel encountered a problem while loading
"BriskMenuFactory::BriskMenu".

The panel encountered a problem while loading
"MultiLoadAppletFactory::MultiLoadApplet".

The panel encountered a problem while loading
"InhibitAppletFactory::InhibitApplet".

I was able to resolve the menu issues by installing additional MATE packages manually. I installed a bunch but I believe the following are the culprits:

`sudo apt install mate-applet-brisk-menu mate-applet-appmenu mate-power-manager*`

Did I get them? TY

**Comment by PalinuroSec** [comment]
thank you for having reported it, have you installed mate from the parrot-mate/parrot-interface metapackages?

**Comment by MahdiHedhli** [comment]
Yes, it seems the parrot-interface package or parrot-mate package that gets installed with parrot-interface is missing all the mate menu packages until a manual apt-get update is ran. This has been my experience on several ARM/ARM64 based installs.
________________________________

**Comment by GitStoopID** [comment]
Install this inside parrot or in the MacBook terminal?
Ok got it updated through Parrot but still can't connect via WiFi and no idea how it downloaded and installed the updated brisk menu pkgs when it states its connected to a wired connection but there is none 🤷🏻‍♂️? Firefox doesn't work at all....so disappointing ...back to Kali I suppose....
